plugins/FileHandlers/STLReader/STLReader.py

```diff
@@ -15,23 +14,8 @@
 from UM.Mesh.MeshReader import MeshReader
 from UM.MimeTypeDatabase import MimeTypeDatabase, MimeType
 from UM.Scene.SceneNode import SceneNode
-
-use_numpystl = False
-
-try:
-    # Work around for CURA-7154. Some models crash with a segfault, but only when packed.
-    if platform.system() != "Linux":
-        import stl  # numpy-stl lib
-        import stl.mesh
-
-        # Increase max count. (100 million should be okay-ish)
-        stl.stl.MAX_COUNT = 100000000
```

Comment:
We should still have the increased maximum vertex count. Otherwise it can't open meshes with more than 10 million vertices.